kubernetes/kfserving/kf_predictor_docker/Dockerfile_kf.dev

```diff
@@ -56,8 +56,6 @@ RUN if [ "$MACHINE_TYPE" = "gpu" ]; then export USE_CUDA=1; fi \
     && git checkout ${BRANCH_NAME} \
     && python ts_scripts/install_dependencies.py --environment=dev --cuda $CUDA_VERSION \
     && python ts_scripts/install_from_src.py \
-    && pip install captum \
```

Comment:
removed these statements as these are installed as part of install_depenencies.py

the packages are already included in the common.txt and developer.txt requirements file.